### PR TITLE
Update getting started guide for hc-scaffold 0.3000.11

### PR DESCRIPTION
**Pull request by pdaoust**
Lots of screenshots, code, and interactive scaffolding prompts got changed. Also:

* diff + language highlighting, so diffs aren't just plain Christmas colours (cuz I'm now using diffs in the getting started guide)
* rearranged packaging/deployment tools
    * demoted Launcher lower down the page and added note that develpment is on hold
    * prmoted shipyard higher up the page
    * added a blurb about Moss
    * swapped order of electron vs tauri Kangaroos cuz electron is now the recommended one

### CODE REVIEW

File: src/pages/get-started/4-packaging-and-distribution.md [hunk 1]
```diff
@@ -56,39 +56,56 @@ In the centralized world, deployment is usually achieved by Continuous Integrati
...
-
-You can try this latter approach with your forum app immediately by downloading and running the Launcher! If you install `workdir/my_forum_app.webhapp`, the hApp _and its UI_ will be installed.
-
-The steps for publishing an app to the Launcher's app store are documented in the [Github repository of the Holochain Launcher](https://github.com/holochain/launcher#publish-an-app-to-launchers-app-store).
-
-!!!info Launcher uses Electron
-If you're developing an app that's meant to be run within Launcher, it's important to know that Launcher uses Electron, which embeds a fairly recent version of Google's [Blink](https://www.chromium.org/blink/) browser engine. The nice thing about this is that your UI can target a consistent browser, which should shorten development time.
+1. Download your app as its own standalone desktop executable, as they would any other application for their computer.
+2. Download [Moss, an experimental groupware application](https://theweave.social), create a group of people to work with, and install the app as a 'tool' into their group.
+2. Download Holochain's official Launcher runtime and install the app from its app store or the filesystem.
```
Comment (matthme) [inline src/pages/get-started/4-packaging-and-distribution.md:61]:
```suggestion
3. Download Holochain's official Launcher runtime and install the app from its app store or the filesystem.
```

File: src/pages/get-started/4-packaging-and-distribution.md [hunk 2]
```diff
@@ -56,39 +56,56 @@ In the centralized world, deployment is usually achieved by Continuous Integrati
 
 From the end-user perspective there are currently two ways to go about this:
```
Comment (matthme) [inline src/pages/get-started/4-packaging-and-distribution.md:57]:
```suggestion
From the end-user perspective there are currently three ways to go about this:
```

**Comment by pdaoust** [comment]
**Update**: The diff + language highlighting doesn't seem to work. I've create an issue https://github.com/valeriangalliat/highlightjs-code-diff/issues/1 ; we'll see what happens there. For now, it's not harming anything so I'll leave it in.

File: src/pages/get-started/4-packaging-and-distribution.md [hunk 3]
```diff
@@ -56,39 +56,56 @@ In the centralized world, deployment is usually achieved by Continuous Integrati
...
 
-* **Tauri**: See the officially supported [holochain-kangaroo](https://github.com/holochain-apps/holochain-kangaroo) repo. Note that Tauri uses the webview engine provided by the user's operating system, so you may need to do extensive cross-OS testing to make sure your UI works properly.
 * **Electron**: See the officially supported [holochain-kangaroo-electron](https://github.com/holochain-apps/holochain-kangaroo-electron) repo.
```
Comment (matthme) [inline src/pages/get-started/4-packaging-and-distribution.md:67]:
```suggestion
* **Electron**: See the officially supported [holochain-kangaroo-electron](https://github.com/holochain/kangaroo-electron) repo.
```

File: src/pages/get-started/4-packaging-and-distribution.md [hunk 4]
```diff
@@ -56,39 +56,56 @@ In the centralized world, deployment is usually achieved by Continuous Integrati
...
+* **Tauri**: See the officially supported [holochain-kangaroo](https://github.com/holochain-apps/holochain-kangaroo) repo. Note that Tauri uses the webview engine provided by the user's operating system, so you may need to do extensive cross-OS testing to make sure your UI works properly.
```
Comment (matthme) [inline src/pages/get-started/4-packaging-and-distribution.md:68]:
```suggestion
* **Tauri**: See the officially supported [holochain-kangaroo](https://github.com/holochain-apps/kangaroo-tauri) repo. Note that Tauri uses the webview engine provided by the user's operating system, so you may need to do extensive cross-OS testing to make sure your UI works properly.
```

**Review by matthme — CHANGES_REQUESTED** [review]
A few smaller changes requested.

File: src/pages/get-started/4-packaging-and-distribution.md (hunk 3 above)
Comment (matthme) [reply src/pages/get-started/4-packaging-and-distribution.md:67]: maybe also rename `holochain-kangaroo-electron` to `kangaroo-electron`?

File: src/pages/get-started/4-packaging-and-distribution.md (hunk 4 above)
Comment (matthme) [reply src/pages/get-started/4-packaging-and-distribution.md:68]: And similarly here `holochain-kangaroo` to `kangaroo-tauri`

File: src/pages/get-started/4-packaging-and-distribution.md (hunk 1 above)
Comment (pdaoust) [reply src/pages/get-started/4-packaging-and-distribution.md:61]: I always number things 1, 2, 2. Just my personal style.

J/K; thanks for catching that.

**Review by matthme — APPROVED** [review]
Looks good to me now!

**Pull request merged**